R/calculate_annual_pd_changes.R

```diff
@@ -17,10 +17,15 @@
 #'   which year the analysis runs
 #' @param risk_free_interest_rate A numeric vector of length one that indicates
 #'   the risk free rate of interest
+#' @param exposure_at_default A dataframe that contains the share of the
+#'   portfolio value of each company-technology combination. Used to quantify
+#'   the impact of the company-tech level shock on higher levels of aggregation
+#'   in the portfolio
 calculate_pd_change_annual <- function(data,
                                        shock_year = NULL,
                                        end_of_analysis = NULL,
-                                       risk_free_interest_rate = NULL) {
+                                       risk_free_interest_rate = NULL,
+                                       exposure_at_default = NULL) {
```

Comment:
no NULL default please. Instead move it to second position as an obligatory arg please

Reply:
done